cartographer_ros_msgs/msg/TrajectoryOptions.msg

```diff
@@ -18,6 +18,7 @@ string odom_frame
 bool provide_odom_frame
 bool use_odometry
 bool use_nav_sat
+bool project_pose_to_2d_plane
```

Comment:
rename to `publish_frame_projected_to_2d`, also in all lua configs and the struct

Reply:
@cschuet Do you agree with this name?